lib/brcobranca/boleto/template/rghost.rb

```diff
@@ -177,8 +177,10 @@ def modelo_generico_rodape(doc, boleto)
           doc.show boleto.local_pagamento
           doc.moveto x: '16.5 cm', y: '16 cm'
           doc.show boleto.data_vencimento.to_s_br if boleto.data_vencimento
-          doc.moveto x: '0.7 cm', y: '15.2 cm'
+          doc.moveto x: '1.9 cm', y: '15.5 cm'
           doc.show boleto.cedente
+          doc.moveto x: '0.7 cm', y: '15.2 cm'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.